library/ibm_is_instance_action.py

```diff
@@ -69,7 +72,25 @@ def run_module():
     service = VpcV1('2020-06-02', authenticator=authenticator)
 
     instanceId = module.params["instance_id"]
-    if module.params["instance_ip"]:
+    if len(module.params["instance_ip_list"]) > 0:
+        try:
+            floatingIps = service.list_floating_ips().get_result()['floating_ips']
+            instances = service.list_instances().get_result()['instances']
+        except ApiException as e:
+            print("List instances/floatingIp failed with status code " + str(e.code) + ": " + e.message)
+            module.fail_json(msg=("List instances/floatingIp failed with status code " + str(e.code) + ": " + e.message))
+
+        msg = ""
+        for ip in module.params["instance_ip_list"]:
+            (res, rmsg) = stopInstance(service, floatingIps, instances, ip, module.params["action_type"])
```

Comment:
this will handle both start/stop ?

Reply:
Yes, it will handle all actions - start, stop & reboot..